<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormCorrectorTuning</class>
 <widget class="QMainWindow" name="FormCorrectorTuning">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>141</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Corrector Tuning</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="titleGroupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>360</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
	background-color: rgb(255, 255, 255);
    border: 1px solid black;
    border-radius: 5px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="btnClose">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Parameter Address</string>
    </property>
   </widget>
   <widget class="QEComboBox" name="cbSetAddress">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="variable" stdset="0">
     <string>SRC01-PS-HC1:setParameterAddress</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Parameter Data</string>
    </property>
   </widget>
   <widget class="QELineEdit" name="txtData">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="variable" stdset="0">
     <string>SRC01-PS-HC1:setParameterData</string>
    </property>
   </widget>
   <widget class="QELabel" name="lblData">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="variable" stdset="0">
     <string>SRC01-PS-HC1:getParameterData</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QEComboBox</class>
   <extends>QComboBox</extends>
   <header>QEComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>QELabel</class>
   <extends>QLabel</extends>
   <header>QELabel.h</header>
  </customwidget>
  <customwidget>
   <class>QELineEdit</class>
   <extends>QLineEdit</extends>
   <header>QELineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
